### ISSUE TITLE
Memory Leak ?

### ISSUE THREAD

**Issue by bcaillabet** [WedjaaOpen/ElasticJasperServer#8]
Hi,

I designed a report with the ElasticJasper connector. I use both ElasticJasper and ElasticJasperServer. My report is about 40 pages with simple requests . When jasperserver build the report, memory usage is increasing normally but not released when the report is generated .

I must restart jasperserver for each generation. The problem seems to be present on both ElasticJasper and ElasticJasperServer.

Benoît

**Comment by Wedjaa** [comment]
Leave the server running - JVM will perform a GC (garbage collection) when it needs more memory. Let me know if running multiple times the report you get an OOM error - that would indicate something wrong with object disposal.

**Comment by bcaillabet** [comment]
Yes running multiple times the report in jasper server finish with this : 

Error Message
java.lang.OutOfMemoryError: Direct buffer memory

here is the full trace :
[OEMtrace.txt](https://github.com/WedjaaOpen/ElasticJasperServer/files/594853/OEMtrace.txt)
